File: 0338-counting-bits/0338-counting-bits.py
class Solution:
    def countBits(self, n: int) -> List[int]:
        l=[]
        for i in range(n+1):
            c=0
            num=i
            while(num):
                if(num%2==1):
                    c+=1
                num//=2
            l.append(c)
        return l

File: 0338-counting-bits/0338-counting-bits.cpp
class Solution {
public:
    vector<int> countBits(int n) {
        vector<int>setss;
        for (int i=0;i<=n;i++){
            int count=0;
            int num=i;
            while(num){
                if(num%2==1){
                    count++;
                }
                num/=2;
                
            }
            setss.push_back(count);
        }
        return setss;
    }
};